### DISCUSSION TITLE
Chinese Translation of MEEP Document

### DISCUSSION THREAD

**Discussion by Fengkai-Wei** [NanoComp/meep#2890] [General]
Hello, I am a active MEEP user for FDTD simulation. Does the team have any plan for Chinese translation? If yes, I would like to participate.

**Comment by stevengj** [comment]
Currently, we have no plans to organize translations of the documentation; we don't really have the resources to maintain this kind of thing.   It would be especially hard for us to ensure that the translated documentation remains up-to-date as Meep evolves.

**Comment by oskooi** [comment]
Translating the entire user manual on readthedocs would be a lot of work. It might be useful to select a small subset of the content. If you would like to work on this on your own, I would suggest starting with the [main page](https://meep.readthedocs.io/en/latest/) and [Introduction](https://meep.readthedocs.io/en/latest/Introduction/).
